### ISSUE TITLE
We need an logo!

### ISSUE THREAD

**Issue by theburningmonk** [getndazn/dazn-lambda-powertools#100]
# This is a Feature Proposal

## Description

A snappy logo for this project.

**Comment by peakgeek** [comment]
I'd be happy to attempt a logo for you. I have an idea brewing already, but wondered if you had any firm expectations, guidelines, or requirements?

Thanks.

**Comment by theburningmonk** [comment]
@peakgeek don't have anything in mind.

@simontabor @gsingh1 can you check if there are any DAZN guidelines/requirements that we need to adhere to here?

**Comment by peakgeek** [comment]
Waiting to hear if there are any official guidelines, but this is where my idea was headed. Let me know if it's even in the ballpark of what you're looking for. My working file is a vector image that can be tweaked easily.

![logo-v1-512px](https://user-images.githubusercontent.com/3734060/66537594-3be4cb00-eade-11e9-8435-94ce45e0c76c.png)

**Comment by theburningmonk** [comment]
@peakgeek looks good on the whole, but I don't get the reference of the "A" :-P

**Comment by theburningmonk** [comment]
@peakgeek that was not sarcasm, I actually don't understand what the "A" means in the logo.

**Comment by peakgeek** [comment]
Sarcasm is difficult to interpret through text, especially from "internet-strangers," but I'm hoping that's what the "A" reference was. I could try making the Lambda symbol lower-case if that's more your style. Colors were also selected arbitrarily if you have a color scheme you gravitate towards or that is more in line with the project, just let me know!

**Comment by peakgeek** [comment]
It's the greek symbol for Lambda.

**Comment by theburningmonk** [comment]
ahhhh, right, of course! Not used to seeing like that, normally see it like this: λ

I don't have any objections. But let's wait to see if there's any official style guides from DAZN.

**Comment by peakgeek** [comment]
Yeah, sorry, the ":-P" threw me off and I couldn't tell if you were serious or not. I'm honestly not super familiar with your project. I'm just a "graphic design hobbyist" that is trying to participate in Hacktoberfest and thought I could help your team out. If the lambda symbol doesn't make sense or if the DAZN (acronym?) is more significant (or maybe incorporating both symbol and DAZN makes the most sense) I would be happy to try and assist with this. I was just stabbing in the dark on attempt one.
